tests/performance/config/perf-driver/policies/test-n-apps-1-instances.yml

```diff
@@ -48,9 +48,7 @@ policies:
       - name: Scale Apps
         values:
           - parameter: apps
-            min: 50
-            step: 50
-            max: 500
+            values: [100, 1000]
```

Comment:
Again, didn't we agree on a single value/run here and above: `10.000`?

Reply:
Again, this particular test has 2 points, and will keep having 2 data points. 
**This** is the test that has a single data point : https://github.com/mesosphere/marathon/pull/6300

To be more precise, this test should sample 100, 1k and 10k. However there are issues deploying 10k apps on the v1.5.9 (reference), that I am trying to solve elegantly. Since this process is taking too long, I vote for merging this test with the 2 data points (100, 1k) so we can unblock #6300 , and then I can add the 10k data point when I have a solution.

Reply:
> This is the test that has a single data point : #6300
it looks like it has 3 datapoints to me ;)